ui/components/multichain/asset-picker-amount/asset-picker-modal/asset-picker-modal.tsx

```diff
@@ -128,12 +128,14 @@ export function AssetPickerModal({
           event: MetaMetricsEventName.sendAssetSelected,
           category: MetaMetricsEventCategory.Send,
           properties: {
-            ...sendAnalytics,
             is_destination_asset_picker_modal: Boolean(isDest),
             new_asset_symbol: token.symbol,
             new_asset_address: token.address,
```

Comment:
Another reference here to move into `sensitiveProperties`

Reply:
Good call - fixed